.github/workflows/build_worker_images.yml

```diff
@@ -0,0 +1,38 @@
+name: Build default worker images
+on:
+  push:
+    branches: [master]
+    paths: ["docker_config/dockerfiles/default-*"]
+  release:
+    types: [published]
+    paths: ["docker_config/dockerfiles/default-*"]
+  pull_request:
+    paths: ["docker_config/dockerfiles/default-*"]
+
+jobs:
+  build_default:
+    name: Build and push default worker images
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        service: [default-cpu, default-gpu]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: pip-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            pip-
+      - run: pip install -r requirements.txt
+      - run: python3 codalab_service.py build ${SERVICE} $([ -z "${CODALAB_DOCKER_USERNAME}" ] || echo "--push")
+        env:
+          CODALAB_DOCKER_USERNAME: ${{ secrets.CODALAB_DOCKER_USERNAME }}
+          CODALAB_DOCKER_PASSWORD: ${{ secrets.CODALAB_DOCKER_PASSWORD }}
+          # Gives us the branch name of the PR if on a pull_request-triggered build,
+          # otherwise, "master" if on a push-triggered build
+          VERSION: ${{ github.head_ref || 'master' }}
```

Comment:
We need to ensure we run an additional build step in which we set VERSION to "latest" when this build is triggered by a release.